### PR TITLE
This commit is an addition to the executable screen of pip uninstall 

### PR DESCRIPTION
**Pull request by yhs061017**
install in venv.rst

----
:books: Documentation preview :books:: https://cpython-previews--110228.org.readthedocs.build/

### CODE REVIEW

File: Doc/tutorial/venv.rst [hunk 1]
```diff
@@ -146,6 +146,16 @@ different version number to get that version, or you can run ``python
 ``python -m pip uninstall`` followed by one or more package names will
 remove the packages from the virtual environment.
 
+.. code-block:: bash
```
Comment (AA-Turner) [inline Doc/tutorial/venv.rst:149]:
```suggestion
.. code-block:: bash

```

**Review by AA-Turner — COMMENTED** [review]
I'm not sure this is worth it, please could you provide some rationale for the change?

A

**Comment by yhs061017** [comment]
It's my first time contributing to open source So I wanted to speak in Python, my favorite language I wanted to show the results of the pip uninstall on venv.rst I'm very sorry if it interfered with the Python project. Thank you for your interest in my small activities

**Comment by vsajip** [comment]
> I'm very sorry if it interfered with the Python project

It's not a question of "interference", just a request for you to justify why this is really needed. It doesn't seem add any significantly useful information, and if you add this example and a user follows it step-by-step, then it won;t make sense because the next example (`python -m pip show requests`) won't work if you've uninstalled `requests`.

**Comment by AA-Turner** [comment]
I'll close this PR, I agree with Vinay's reasoning, and I think that the information about uninstalling is presented here for information/symmetry rather than directing the reader to immediately uninstall the package that was just installed.

A

**Pull request closed without merging**